### PR TITLE
Render results grid

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: controller/ClientdiagTool.php [hunk 1]
```diff
@@ -33,6 +32,10 @@ class ClientdiagTool extends \taoLti_actions_ToolModule
      */
     public function run()
     {
-        
+        if ($this->hasAccess(Diagnostic::class, 'index')) {
+            $this->redirect(_url('index', 'Diagnostic'));
+        } else {
+            $this->returnError('You are not authorized to access this resource');
```
Comment (siwane) [inline controller/ClientdiagTool.php]: Could u transalte it please

File: scripts/install/ltiroles.rdf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<rdf:RDF 
+  xml:base="http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#"
+  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+>
+  
+  <!-- Context roles -->
+  <rdf:Description rdf:about="http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#TeachingAssistant">
```
Comment (siwane) [inline scripts/install/ltiroles.rdf]: TeacherAssistant is not used, right?

File: manifest.php [hunk 3]
```diff
@@ -18,32 +18,34 @@
...
         'taoLti' => '>=1.7.1'
-    ),
-    'acl' => array(
-
-    ),
-    'install' => array(
+    ],
+    'managementRole' => 'http://www.tao.lu/Ontologies/generis.rdf#ltiClientdiagManager',
+    'acl' => [
+        array('grant', 'http://www.tao.lu/Ontologies/generis.rdf#ltiClientdiagManager', array('ext'=>'ltiClientdiag')),
```
Comment (siwane) [inline manifest.php]: Could u put it on 'role interface const' please

Comment (siwane) [inline manifest.php]: Could u avoid tu use a backOffice role to access this ext?

File: scripts/install/ltiroles.rdf (hunk 2 above)
Comment (hutnikau) [inline scripts/install/ltiroles.rdf:9]: It used in lti. Actually this is LTI role.

File: manifest.php [hunk 4]
```diff
@@ -18,32 +18,34 @@
...
+return [
     'name' => 'ltiClientdiag',
     'label' => 'LTI Client Diagnostic',
     'description' => 'Grants access to the client diagnostic functionality using LTI',
     'license' => 'GPL-2.0',
     'version' => '0.0.1',
     'author' => 'Open Assessment Technologies SA',
-    'requires' => array(
+    'requires' => [
         'taoLti' => '>=1.7.1'
```
Comment (siwane) [inline manifest.php]: Could u require taoCLientDiag?

File: manifest.php (hunk 3 above)
Comment (siwane) [inline manifest.php]: ok

**Pull request merged**